src/index.js

```diff
@@ -1,3 +1,6 @@
+// Package metadata
+var pkg = require('../package');
+
```

Comment:
Necessary to hoist this because the `file:` check uses it. (You asked me to correct the version number, I made it dynamic).